Core/src/Upload/ResumableUploader.php

```diff
@@ -274,12 +274,13 @@ protected function getStatusResponse()
      * Gets the starting range for the upload.
      *
      * @param string $rangeHeader
-     * @return int|null
+     * @return int
      */
     protected function getRangeStart($rangeHeader)
     {
         if (!$rangeHeader) {
-            return null;
+            // assume no bytes are uploaded if no range header is present
+            return 0;
         }
```

Comment:
What situation leads to the `rangeHeader` becomes empty? 

As per the [documentation](https://cloud.google.com/storage/docs/performing-resumable-uploads#chunked-upload), I see that we should get Range header when 308 is the response code.

Reply:
All content range queries on resume uri should be responded with 308 until upload is completed for all bytes.
This particular case, we are seeing a 308 response but it has an empty header. Because no data has been uploaded so far, so we want to resume upload from zero'th byte.